### PR TITLE
Convert to 0.12 syntax

### PR DESCRIPTION
**Pull request by ribbybibby**

### CODE REVIEW

File: cfssl.tf [hunk 1]
```diff
@@ -7,15 +7,15 @@ resource "google_service_account" "cfssl" {
...
-  zone = "${var.available_zones[0]}"
+  zone = var.available_zones[0]
   size = 5
   type = "pd-standard"
 
   // The value can only contain lowercase letters, numeric characters, underscores and dashes
-  labels {
-    name      = "cfssl-${var.cluster_name}-data-vol-${count.index}"
+  labels = {
+    name      = "cfssl-${var.cluster_name}-data-vol-0"
```
Comment (hectorhuertas) [inline cfssl.tf]: should this be left as count.index?

Comment (ribbybibby) [reply cfssl.tf]: This resource doesn't actually have count set, so it returns an error in 0.12. I'm not sure if there was a particular reason we used `count.index` in the first place but it doesn't work anymore.

Comment (ribbybibby) [reply cfssl.tf]: I suppose we could set `count = 1` and this would work. But if we only need one resource, we only need one.

Comment (hectorhuertas) [reply cfssl.tf]: yeah, I guess it doesn't matter anymore

**Pull request merged**